.github/workflows/deploy.yml

```diff
@@ -39,32 +39,20 @@ jobs:
   pypi:
     strategy:
       matrix:
-        distribution: [sdist, bdist_wheel]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        distribution: [bdist_wheel]
+        python-3-version: [6, 7, 8, 9]
         include:
-          - python-version: 3.6
-            wheel-tag: py36
-          - python-version: 3.7
-            wheel-tag: py37
-          - python-version: 3.8
-            wheel-tag: py38
-          - python-version: 3.9
-            wheel-tag: py39
-        exclude:
           - distribution: sdist
```

Comment:
SDist job runs here?

Reply:
Might be easier to look at the branch than the diff, that's a little messy the way it's split up. Or use the "split" view mode in GitHub.

Reply:
Yeah, I see that now. Thanks for pointing this out.